### ISSUE TITLE
Server gets stuck in infinite loop in BettingRound

### ISSUE THREAD

**Issue by jezdawgz** [jezdawgz/Poker#2]
Server gets stuck in infinite loop while polling for current Player Event in BettingRound.

```java
while (evt == null && current.isConnectionAlive()) {                    
     evt = current.readEvent();
}    
```

**Comment by jezdawgz** [comment]
Add a ```Thread.sleep(long)``` call to solve the issue

**Comment by jezdawgz** [comment]
Fixed in 0cfcf57f13e2d77eacb6d5ac6f5a4aa2899dfe14
